CHANGES.md

```diff
@@ -1,4 +1,6 @@
-# 2.0.2 (UNRELEASED)
+# 2.0.2 (2018-02-14)
+- [NOTE] Updated `require` references to use scoped package
```

Comment:
Should this be a `[FIXED]` since it corrects the examples and documentation?

If not I think it should be re-worded slightly to something like `Corrected ... to use new scoped ...`